src/App.tsx

```diff
@@ -1,3 +1,45 @@
+import { useState } from 'react';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+
+import { Footer } from './components/Footer';
+import { Navbar } from './components/Navbar';
+import { DataContextProvider } from './hooks/DataContext';
+import { BrowseErrors } from './pages/BrowseErrors';
+import { StructureExplorer } from './pages/StructureExplorer';
+
 export default function App() {
-  return <h1 className="text-3xl font-bold underline">Hello world!</h1>;
+  const [showAbout, setShowAbout] = useState(false);
+
+  return (
+    <div className="min-h-screen bg-lightgray font-Archivo font-medium tracking-wide">
+      <Navbar setShowAbout={setShowAbout} />
+      <DataContextProvider>
+        <BrowserRouter>
```

Comment:
Can you use HashRouter? We deploy this site on GitHub pages, where we cannot have all URLs point to index.html